src/components/Header/Header.module.scss

```diff
@@ -14,22 +14,22 @@
   padding-bottom: 1.5em;
 }
 
-.Header-Separator {
```

Comment:
I like the naming convention but just want to know how so the color and added variables are gone? were they gotten from a parent?